compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -280,8 +280,12 @@ OMR::Power::CodeGenerator::initialize()
 
    // Standard allows only offset multiple of 4
    // TODO: either improves the query to be size-based or gets the offset into a register
-   if (comp->target().is64Bit())
+   if (!comp->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P9) &&
+       !(comp->target().cpu.isBigEndian() && comp->target().cpu.is(OMR_PROCESSOR_PPC_P8))
+      )
```

Comment:
Minor formatting nit:

```suggestion
   if (!comp->target().cpu.isAtLeast(OMR_PROCESSOR_PPC_P9) &&
       !(comp->target().cpu.isBigEndian() && comp->target().cpu.is(OMR_PROCESSOR_PPC_P8)))
```

Reply:
I'll make this change as well.